doc/api/dgram.md

```diff
@@ -108,6 +108,9 @@ Tells the kernel to join a multicast group at the given `multicastAddress` and
 one interface and will add membership to it. To add membership to every
 available interface, call `addMembership` multiple times, once per interface.
 
+When called on an unbound socket, implicitly binds to a random port, listening
+on all interfaces.
```

Comment:
This kind of sentence fragment (subject is unstated) is used in the first sentence in an entry, but subsequently, we identify the subject.

```suggestion
When called on an unbound socket, it will implicitly bind to a random port,
listening on all interfaces.
```